{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:189:1:\n    |\n189 | import \"../IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/utils/Address.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:190:1:\n    |\n190 | import \"../../../utils/Address.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:558:1:\n    |\n558 | import \"@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/security/ReentrancyGuard.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:559:1:\n    |\n559 | import \"@openzeppelin/contracts/security/ReentrancyGuard.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/utils/cryptography/MerkleProof.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:560:1:\n    |\n560 | import \"@openzeppelin/contracts/utils/cryptography/MerkleProof.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/00/000047203100A45635029eC21bbBec5EC53Cb6f6_SaleManager.sol:561:1:\n    |\n561 | import \"@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
